client/Hooks/README.md

```diff
@@ -13,16 +13,24 @@ For a given Hook `useFoo`, the expected files are as follows:
 ```
 Hooks/
   index.ts
+  types.ts
   useFoo/
     README.md
     Hook.ts
```

Comment:
UseFoo.*

Reply:
Actually, we should name the hook `Foo`, and then it's called `useFoo` when exported?